### PR TITLE
Fix demo for browsers that don't support Object.observe

### PR DESCRIPTION
**Pull request by soonick**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -38,6 +38,7 @@
   selector.addEventListener('change', function() {
     document.documentElement.lang = this.value;
     I18nMsg.lang = this.value;
+    Platform.performMicrotaskCheckpoint();
```
Comment (ebidel) [inline demo/index.html]: can we use `WebComponents.performMicrotaskCheckpoint()` instead?

Comment (soonick) [inline demo/index.html]: I tried to make the change you suggested but I couldn't a method called performMicrotaskCheckpoint inside of the WebComponents object. I also tried searching for performMicrotaskCheckpoint inside webcomponents.js and I couldn't find it.

Comment (ebidel) [inline demo/index.html]: That's weird. It exists in polymer-project.org in the console. WebComponents  was aliased to the old Platform a while ago.

Comment (soonick) [inline demo/index.html]: I think I found the problem. It seems like the mapping(https://github.com/webcomponents/webcomponentsjs/blob/master/src/WebComponents/bc.js) is done for the full version(https://github.com/webcomponents/webcomponentsjs/blob/master/src/WebComponents/build.json) but not for lite version(https://github.com/webcomponents/webcomponentsjs/blob/master/src/WebComponents/build-lite.json)

Should I just use the full version of webcomponents on this file?

Is this something that needs to be fixed in the webcomponents repo? I can create a pull request for that if necessary.

Comment (ebidel) [inline demo/index.html]: Ah actually, this should be unnecessary. Polymer no longer uses `object.observe()`.  `performMicrotaskCheckpoint` was to flush the data system, where the polyfill was used and o.o() wasn't available.

I haven't ran your demo. Is there actually a problem in polyfill'd browsers?

Comment (soonick) [inline demo/index.html]: Yes, it doesn't work on FF without this change.

```js
     var observerLang = new PathObserver(window.I18nMsg, 'lang');
      observerLang.open(function(newValue, oldValue) {
        _numInstancesUpdated = 0;
        this._setLanguage(newValue);
      }.bind(this));
```

Object.observe is being used to watch changes on I18nMsg.lang, which is changed when another  language is selected from the select box.

Maybe another option would be to make **this.setLanguage()** public?

Comment (ebidel) [inline demo/index.html]: Apologies. I was having a senior moment. Polymer no longer uses O.o() or the poyfill, but this component does (observe-js) :)

**Comment by ebidel** [comment]
LGTM after all :)

**Pull request merged**